src/profiler/profiler.h

```diff
@@ -105,8 +114,7 @@ class Profiler
 	~Profiler();
 
 	// DE: 20090325: Profiler no longer owns callstack and flatcounts since it is shared between multipler profilers
-	std::map<CallStack, SAMPLE_TYPE>& callstacks;
-	std::map<PROFILER_ADDR, SAMPLE_TYPE>& flatcounts;

```

Comment:
It would be nice if you could add a description (a few paragraphs) of the changes in this PR in the commit message. For example, it could mention why `flatcounts` were removed, and what replaces them (if anything). This will help future contributors as well.